compiler/env/OMRVMEnv.cpp

```diff
@@ -118,6 +118,12 @@ OMR::VMEnv::getHighResClockResolution(OMR_VMThread *omrVMThread)
    return ::highResClockResolution();
    }
 
+uint64_t 
+OMR::VMEnv::getHighResClockResolution(TR_FrontEnd *fe)
```

Comment:
getHighResClockResolution() with no parameter does not exist. The function needs access to the portlib where the real implementation resides.